Clarify uppercase characters (#16)
ossf/package-manager-best-practices: review/npm.md

   assess its trustworthiness through, for example, its number of contributors,
   stars, etc.

    1. Additionally, uppercase characters were previously allowed, which is another
       attack vector to be aware of. For example: [JSONStream](https://www.npmjs.com/package/JSONStream),
       [jsonstream](https://www.npmjs.com/package/jsonstream).

>     1. Note: Non-ASCII characters are [not supported in npm package

Reviewer: ```suggestion
    1. Note: Non-ASCII characters are [no longer supported in npm package
```

       names](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#name),

>        so developers need not worry about homograph attacks, whereby an attacker

Reviewer: ```suggestion
       so developers need not worry about homograph attacks from the public registry, whereby an attacker
```

       names their package using non-ascii characters that render similar to an

>        ascii character.

Reviewer: ```suggestion
       ascii character. Note that this property is registry-dependent;
       you will need to verify this policy with any registry you use.
```
